src/SeqViz/SeqViz.tsx

```diff
@@ -179,7 +180,7 @@ export default class SeqViz extends React.Component<SeqVizProps, any> {
   /**
    * Search for the query sequence in the part sequence, set in state
    */
-  search = (part: Part | null = null) => {
+  _search = (part: Part | null = null) => {
```

Comment:
what is this for? Looks like python w/ the underscore to indicate a private method. Re: google style guide link below lets avoid the underscore prefix

Reply:
This was an attempt to distinguish from the local `search` var created inside the search method - it was a little confusing keep the method, the state var, and the props var straight. I'll rename it back unless you have an alternate name you prefer.